requirements-focal.txt

```diff
@@ -0,0 +1 @@
+http://archive.ubuntu.com/ubuntu/pool/main/p/python-apt/python-apt_2.0.0ubuntu0.20.04.5.tar.xz; sys_platform == 'linux'
```

Comment:
IMO, I wouldn't pin this req at all.  Depend on the host for it (python3-apt).

Reply:
Separately if craft-parts is to support non-Ubuntu bases, it shouldn't be a hard requirement either unless on deb/ubuntu.

Reply:
Using python3-apt should make things easier.

Reply:
this is requirements-focal.txt, so not really about not Ubuntu. When we support other distros the "import apt" should be lazy loaded,  catched and set to a stub for the non compatible platforms, or we can build the apt libs ourselves (at the application level).

python3-apt from debs would require setting the venv to also look at dist-packages.

There's a slight clarification to be made here too, it is not that python-apt is tied to a given Ubuntu libapt but to a given version of the library, if we built it, it could move out of band from the base we use (for the snap case or any other foreign os case)